### PR TITLE
Feature/backgrounds

### PR DESCRIPTION
**Pull request by CLPena**
We resolved conflicts together so this should all work. It doesn't yet include Jeremiah's save form functionality.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ function addGarment(event) {
   toggleGarments('hat', 0);
   toggleGarments('clothes', 1);
   toggleGarments('accessories', 2);
-  toggleGarments('backgrounds', 3);
```
Comment (hborrelli1) [inline main.js]: This line is no longer necessary, right?

**Comment by jeremiahblackol** [comment]
you did that much cleaner than i would have

File: outfit.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ class Outfit {
   constructor(id){
     this.id = id;
     this.title = '';
-    this.garments = [null, null, null, null];
+    this.garments = [null, null, null];
```
Comment (hborrelli1) [inline outfit.js:5]: Good call, we also don't need that extra space anymore since the backgrounds have their own property.

File: main.js (hunk 1 above)
Comment (CLPena) [reply main.js]: Good point. We'll delete it in our next clean-up.

**Pull request merged**